spring-cloud-stream/src/test/java/org/springframework/cloud/stream/binder/PollableConsumerTests.java

```diff
@@ -63,7 +64,6 @@
 
 	@Before
 	public void before() {
-		//System.setProperty("spring.main.allow-bean-definition-overriding", "true");
```

Comment:
I believe this must be removed.
Well, the whole change to this class can be reverted.
Although I would leave an empty line after package 😉